online-resources/learn-to-code-resources.md

```diff
@@ -4,9 +4,12 @@
 
 * [Code First Girls: Teach Yourself](https://www.codefirstgirls.org.uk/teach-yourself.html)
 * [Free Code Camp](https://learn.freecodecamp.org/)
+* [The Odin Project](https://www.theodinproject.com/home)
 * [Exercism.io](http://exercism.io/)
 * [Codewars](https://www.codewars.com/)
 * [Codebar Tutorials](http://tutorials.codebar.io/)
+* [Khan Academy - Computing](https://www.khanacademy.org/computing)
```

Comment:
Interesting thanks, so I have both these in another link but I think this shows I should extract out all the links & add them here so it's a one stop shop & people don't have to follow links. Would you agree?

Reply:
Yes, I would have expected these here - it's where I'd go if I want to find more information. I leave it up to you to decide where you want to put them in the long run :)